client/extensions/zoninator/components/settings/zone-creator/index.jsx

```diff
@@ -1,5 +1,43 @@
-const ZoneCreator = () => {
-	return null;
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import page from 'page';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { flowRight } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import HeaderCake from 'components/header-cake';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+
+const ZoneCreator = ( { siteSlug, translate } ) => {
+	const handleGoBack = () => {
+		page( `/extensions/zoninator/${ siteSlug }` );
+	};
```

Comment:
Hrm, do we really need to specify an `onClick` here? Making that a function that just calls `page` is quite inelegant. Can't we specify an `href` or something? https://github.com/Automattic/wp-calypso/tree/a70587157b33783ba4d5e0e4a1e136680d459d51/client/components/header-cake says there's `backHref`, let's maybe try that. (It also says `onClick` is required :roll_eyes: but let's give it a go.)